app/models/google/calendar/calendar.rb

```diff
@@ -129,8 +140,24 @@ def summary
       event_hash['summary']
     end
 
+    def description
+      event_hash['description']
+    end
+
     def event_link
       event_hash['htmlLink']
     end
+
+    def attendees
+      event_hash['attendees'][0].email if event_hash['attendees'][0]
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.